standards/SPIFFE_Federation.md

```diff
@@ -307,6 +307,56 @@ As described in the [Trustworthiness of the Bundle Endpoint Server](#74-trustwor
+
+In considering how to process claims made by foreign trust domains, it is helpful to consider three categories of claims:
+
+  1. Claims required by and/or defined by the SPIFFE specifications for the SVID type
+  2. Claims that are standardized by the relevant standards body for the document format (i.e. ITU-T for X.509, IETF for JWT)
+  3. Non-standardized claims
+
+Claims in the first category are well-defined in their meaning, and in many cases implementations are required to process them to adhere to SPIFFE specifications.
+
+Claims in the second category have standardized meaning, but operators have some choice whether to and how to process them.
+
+Claims in the third category are not standarized, so operators need to consider them on a case-by-case basis, possibly including navigating different meanings by different trust domains. Needless to say, this category should be approached with the highest degree of caution.
+
+#### 7.7.2 Claim scope
+
+Some claims, like the SPIFFE ID are _explicitly_ scoped to a trust domain by their structure. Because SVIDs are always issued by a particular trust domain authority, _all_ the claims therein are _implicitly_ scoped to that trust domain. This scoping should be preserved when presenting claims to any decision-making process.
+
+As a hypothetical example, consider a claim about the version number of the control plane software that generated the SVID. Different trust domain control planes might use entirely different software packages, and as such the version numbers are not directly comparable in an unscoped context.
+
+
+#### 7.7.3 Interpretation of claims
+
+If claims are to be processed in making a security decision (such as whether to allow a request or connection), it is critical that the authority issuing the claim has the same meaning or interpretation of the claim as the entity making the decision.
+
+In considering the categories of claims noted in [Section 7.7.1](#771-svid-claims), these should be treated with an increasing level of scrutiny around interpretation.
+
+Also consider exactly who is operating the trust domain making the claim. Trust domain operators may be able to agree on the meaning of a particular claim. For example, if an organization operates multiple trust domains for administrative reasons, it may be straightforward to agree on an interpretation of claims within that organization (even if the claims are not publically standardized). The operator should, however, limit processing of that claim only to the trust domains with which the meaning has been agreed.
+
+#### 7.7.4 Veracity of claims
+
+Just because a trust domain authority asserts a claim does not guarantee the claim is true. When choosing whether to accept particular claims from particular trust domains, operators may wish to conduct audits, ask for evidence, or otherwise investigate the processes by which the trust domain makes the claim.
```

Comment:
```suggestion
Just because a trust domain authority asserts a claim does not guarantee the claim is true at time of evaluation or was true at the time that the authority asserted the claim. When choosing whether to accept particular claims from particular trust domains, operators may wish to conduct audits, ask for evidence, or otherwise investigate the processes by which the trust domain makes the claim.
```

Reply:
@justinburke can you talk about why it's important to highlight the difference between the question "is the claim true" and "was the claim true when it was asserted"?  Like, is there some specific concern about the temporal question with relation to foreign trust domains?

Reply:
This is exactly one of the sharp edges that drives me (personally) to discourage folks from embedding subject claims in an SVID. Often times, the claims that are asserted are not natural properties, and as such they can change. This has famously been a challenge for human-oriented X.509 auth wherein security clearance, team membership, etc is embedded and used for authorization. For these kinds of details, it is always better (IMO) to delegate the lookup to a secondary system.

I can see a few ways that this issue occurs for workloads too. One contrived example could be an assertion that the workload is running in/on a specific location or host, but the key for the SVID was not bound to the host and the workload has since been live migrated elsewhere.

Reply:
@evan2645 what you're talking about seems to be a sharp edge related to embedding extra claims at all.  That is to say, it's not a special concern related to the fact that you're interacting with a foreign trust domain.

I think a security considerations discussion of those kinds of issues fits more naturally in one or both of the SVID specs.

Reply:
This section is discussing whether the foreign trust domain can be trusted to tell the truth.

Introducing the concept of the time delay between issuance and evaluation here, without actually developing it, just serves to muddy the waters, rather than clarify.  I agree this is an important security consideration, but it really needs its own section where it can be given the attention it deserves.